spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1,5 +1,8925 @@
+:sectnums!:
+== Preface
+
+This is the Jakarta Server Pages™ Standard Tag
+Library 2.0 (JSTL 2.0) specification, developed by the JSR-52 expert
+group under the Java Community Process. +
+See http://www.jcp.org.
+
+== Related Documentation
+
+Implementors of JSTL and authors of JSP pages
+may find the following documents worth consulting for additional
+information:.
+
+[width="100%",cols="50%,50%",]
+|===
+|Jakarta Server Pages (JSP)
```

Comment:
Remove (JSP), this is a term we agreed not to use anymore.

Reply:
In the actual Jakarta Server Pages specification we actually don't use JSP any place? I talked with @kwsutter  and as long as we recognize it as an Oracle TM on the first use we can use the acronyms if we want. I'm not sure what other specs have done thus far, There are probably a hundred or more occurrences of "JSP" in the spec. Still think we should spell it out each time?

Reply:
I did replace it everywhere in the specs I converted, but indeed, if you feel comfortable with saying something like: "References in this document to JSP refer to the Jakarta Server Pages unless otherwise noted." then it should at least technically be okay.

Reply:
We have a lot of instances of `JSP pages` currently. Thoughts on how to convert that:

Reply:
We had the same problem with EJB beans and JTA transaction, and a little with EL expression. In Authorization I used for instance:

`descriptor that identifies the Jakarta Enterprise Bean in whose context the permission is being evalutated.` so that would be equivalent to the version without the double pages.